### DISCUSSION TITLE
Windows Setup and Configuration

### DISCUSSION THREAD

**Discussion by kubilus1** [kubilus1/autoortho#4] [General]
Place to discuss using Autoortho on Windows.

**Comment by sbbasile** [comment]
Bonjour,
J'ai un souci d'installation et de configuration. Je ne comprends pas le tuto. Pourriez-vous expliquer plus précisément l'installation et la configuration s'il vous plait?

**Comment by ekc007** [comment]
Hi,
Can someone help me how this new GUI and setup works (with X-Plane 12)?

Especially I have a few questions:

- Do I have to create the junctions "Earth nav data" and "terrain" under "X-Plane 12/Custom Scenery/z_autoortho" to my root first manually? (My root dir is C:\Autoortho)
- I then started the GUI and clicked run, but it always seems to specify the root dir as "X-Plane 12/Custom Scenery/z_autoortho/_textures". Modifying the config file doesn't seem to change anything.

Thank you very much!

**Reply by kubilus1** [reply]
You don't need to create any directories, the program will set these up for you.  That 'root dir' is correct.

All you should need to do, is specify the 'Custom Scenery' directory, download any regions you wish from the 'Scenery' tab, and hit 'Run'
